src/OpalCompiler-Core/CompilationContext.class.st

```diff
@@ -498,6 +499,16 @@ CompilationContext >> interactive: anObject [
 	interactive := anObject
 ]
 
+{ #category : #accessing }
+CompilationContext >> logDoItEvaluation: aSourceString [ 
+	self logged ifFalse: [ ^self ].
+	
+	Smalltalk globals 
+			at: #SystemAnnouncer 
```

Comment:
This hidden dependency looks strange. I know it was there before...

Reply:
I agree but it was like that before. I just moved it into CompilationContext from OpalCompiler >> logDoIt

Reply:
I agree but it was like that before. I just moved it into CompilationContext from OpalCompiler >> logDoIt